recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   skip: true  # [win]
-  number: 1
```

Comment:
We can probably make this noarch and let the solver fail on Windows due to the lack of `casacore`.